lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -20,6 +20,52 @@ class CancelOutStatus {
 // e.g. (2x^2 * 5) / 2x^2 => 5 / 1
 // Returns a Node.Status object
 function cancelLikeTerms(node) {
+  // Cancel out terms if the term being multiplied
+  // is the same as the denominator
+  if (Node.Type.isOperator(node) && node.op === '*') {
+    let newNode = clone(node);
+    let leftNode = newNode.args[0];
+    let rightNode = newNode.args[1];
```

Comment:
what if the roles you gave left node and right node are reversed? e.g. the right node is a fraction or sum that cancels out with the left node?

also, how do you know there's only two things being multiplied by each other? there can be any number of terms in a multiplication